main.py

```diff
@@ -3,7 +3,18 @@
 
 def main():
     """Função principal que será rodada quando o script for passado para o interpretador."""
-    # COLOQUE SEU CÓDIGO AQUI
+    num1 = float(input('Digite o primeiro número:'))
+    num2 = float(input('Digite o segundo número:'))
+    num3 = float(input('Digite o terceiro número:'))
+    if num1>num2 and num1>num3:
+      maior= num1
+    if num2>num1 and num2>num3:
+      maior = num2
+    if num3>num1 and num3>num2:
+      maior = num3
```

Comment:
Uma outra ideia só para conhecimento:
```suggestion
    maior = num1
    if num2>maior:
      maior= num2
    if num3>maior:
      maior = num3
```
Vou dar merge, mas esse trecho também funciona para o seu programa.